ReplayKitExample/ReplayKitExample/ReplayKitVideoSource.swift

```diff
@@ -83,6 +87,16 @@ class ReplayKitVideoSource: NSObject, TVIVideoCapturer {
     }
 
     func stopCapture() {
+        if let yBuffer = downscaleYPlaneBuffer {
+            free(yBuffer)
+            downscaleYPlaneBuffer = nil
+            downscaleYPlaneSize = 0
+        }
+        if let uvBuffer = downscaleUVPlaneBuffer {
+            free(uvBuffer)
+            downscaleUVPlaneBuffer = nil
+            downscaleUVPlaneSize = 0
+        }
```

Comment:
This is a bit questionable from a thread safety standpoint. In Swift there is no way to return to ReplayKit's video frame dispatch queue, which is where we are allocating in the first place. We could use an NSLock, dispatch_semaphore or other techniques to synchronize usage of this data.